src/meta/node/src/server.rs

```diff
@@ -674,6 +674,10 @@ pub async fn start_service_as_election_leader(
             Duration::from_secs(1),
         ));
         sub_tasks.push(GlobalBarrierManager::start(barrier_manager));
+
+        if env.opts.enable_automatic_parallelism_control {
+            sub_tasks.push(GlobalStreamManager::start(stream_manager));
```

Comment:
Would you mind giving it a more specific name? At first glance, I was confused why there was no "stream manager" running before this PR.